README.md

```diff
@@ -129,6 +129,26 @@ environment variable
 
 DHIS2 should be available at `http://localhost:8080/dhis2`.
 
+
+##### Overriding default values
+
+You can create a local file called `docker-compose.override.yml` and override values from the main `docker-compose.yml`
+file. As an example, you might want to use a different version of the Postgres database and run it on a different port. 
+More extensive documentation of this feature is available [here]https://docs.docker.com/compose/extends/). Using the override
+file you can easily customize values for your local situation.
+
+```yaml
+version: "3.8"
+
+services:   
+  db:
+    image: postgis/postgis:14-3.3-alpine
+    ports:
+      - 127.0.0.1:6432:5432
+```
+

```

Comment:
here as well, one newline between the sections is enough

Reply:
Thanks. Fixed.